pkg/config/toenv_test.go

```diff
@@ -121,35 +121,33 @@ var _ = Describe("os2 config unit tests", func() {
 		})
 		It("converts to env slice installation parameters", func() {
 			c = Config{
-				Data: map[string]interface{}{
+				CloudConfig: map[string]interface{}{
```

Comment:
changed the name to `CloudConfig` since this is only used to produce cloud-config files. If we need, later on, some other type of free form data I'd suggest adding a new `metadata` or similar stanza.